src/uwtools/tests/test_schemas.py

```diff
@@ -90,6 +95,87 @@ def test_schema_chgres_cube_run_dir(chgres_cube_prop):
     assert "88 is not of type 'string'" in errors(88)
 
 
+# esg-grid
+
+
+def test_schema_esg_grid():
+    config = {
+        "execution": {"executable": "esg_grid"},
+        "run_dir": "/tmp",
+    }
+    errors = schema_validator("esg-grid", "properties", "esg_grid")
+    # Basic correctness:
+    assert not errors(config)
+    # Some top-level keys are required:
+    for key in ("execution", "run_dir"):
+        assert f"'{key}' is a required property" in errors(with_del(config, key))
+    # Additional top-level keys are not allowed:
+    assert "Additional properties are not allowed" in errors({**config, "foo": "bar"})
+    # Additional mpi support is not allowed:
+    assert "Additional properties are not allowed ('mpicmd' was unexpected)" in errors(
+        {"execution": {"mpicmd": "srun"}}
+    )
```

Comment:
@christinaholtNOAA @maddenp-noaa Added this test case to confirm `mpi` support is not allowed.

Reply:
Excellent.